luaui/configs/unit_buildmenu_config.lua

```diff
@@ -132,57 +132,45 @@ end
 -- UNIT ORDER ----------------------
 ------------------------------------
 
+-- At the end of this 'UNIT ORDER' section, unitOrder is an array with unitIDs
+-- sorted by their value specified in unitOrderManualOverrideTable. If no
+-- value is specified, the unit will be placed at the end of the array.
 local unitOrder = {}
 local unitOrderManualOverrideTable = VFS.Include("luaui/configs/buildmenu_sorting.lua")
 
-for unitDefID, _ in pairs(UnitDefs) do
-	if unitOrderManualOverrideTable[unitDefID] then
-		unitOrder[unitDefID] = -unitOrderManualOverrideTable[unitDefID]
-	else
-		unitOrder[unitDefID] = 9999999
-	end
+-- Populate unitOrder with identity values.
+local count = 1
+for _, _ in pairs(UnitDefs) do
+	unitOrder[count] = count
```

Comment:
This assumes `unitDefID`s start from 1 and are consecutive. This is correct at the moment, but if it's not too difficult it would be nice not to introduce a reliance on it (arbitrary `unitDefID`s may come in some fairly distant future)

Reply:
Unlike goddamned featureID's which start from 0 :'(

Reply:
Dont' worry about it, it's not that important and just a good-to-have.